src/Context/ToastContext.tsx

```diff
@@ -1,10 +1,23 @@
+import { createContext } from "react";
+import { toast } from "react-toastify";
 
-const ToastContext = () => {
-  return (
-    <>
-      ToastContext
-    </>
-  )
-}
-
-export default ToastContext
+export const ToastContext = createContext(null)
+export function ToastContextProvider(props) {
+    const getToastValue = (type, message) => {
+        return toast[type](message, {
```

Comment:
We can use the toast function directly in our components and provide default values to the ToastContainer.